tests/configurator.test.js

```diff
@@ -0,0 +1,116 @@
+/* global describe, it */
+
+var assert = require('assert'),
+    noArgs = ['', ''], 
+    noEnv = {}, 
+    Configurator = require('../configurator.js');
+
+describe('the Configurator', function() {
+
+  it ('has a default configuration', function() {
+    var config = Configurator.load();
+    assert.equal(8000, config.port);
+  });
+
+  describe('while dealing with env vars', function() {
+    it ('env vars should win over default configuration', function() {
+      var config = Configurator.load({
+            config: { port: 8000 },
+            argv: noArgs,
+            env: { 'RADAR_PORT': 8001 }
+          });
+
+      assert.equal(8001, config.port);
+    });
+
+    it ('should only overwrite the right keys', function() {
+      var config = Configurator.load({
+        config: {port: 8004},
+        env: { 
+          'RADAR_SENTINEL_MASTER_NAME': 'mymaster',
+          'RADAR_SENTINEL_URLS': 'sentinel://localhost:7777'
+        }
+      });
+      assert.equal(8004, config.port);
+      assert.equal('mymaster', config.sentinelMasterName);
+    });
+
+  });
+
+  function describeOptionTest(name, options){
+    describe('option: ' + name, function() {
+      if (options.default) {
+        it('default must be ' + options.expected, function() {
+          var config = Configurator.load({});
```

Comment:
Though much of our test code has
```javascript
it('...')
```

I prefer
```javascript
it ('...')
```

You use both formats, so let's be consistent, preferably with the second type; we can convert the first type to second in the rest of the tests, later on.